panic send on closed channel using remote goval-dictionary

# Environment

## Vuls
vuls 0.4.2 e788e6a5ad6cf91c5c092b24374d514502881d1c

## OS
- Target Server: ubuntu 16.04
- Vuls Server (local mode): ubuntu 16.04

## Go
- Go version: none
I'm using vuls binary directly over host and with docker ( an special image create with ubuntu 16.04 and the vuls' binary )

# Current Output

```
[Jan  4 18:53:51] DEBUG [localhost] HTTP Request to http://vulsdb.d.ip6n.net:2323/packs/ubuntu/16.04/ncurses
[Jan  4 18:53:51] DEBUG [localhost] CVE-2012-0039 is newly detected by OVAL
[Jan  4 18:53:51] DEBUG [localhost] CVE-2011-5325 is newly detected by OVAL
panic: send on closed channel

goroutine 70 [running]:
github.com/future-architect/vuls/oval.getDefsByPackNameViaHTTP.func1(0xc42052a000, 0xc4202624e0)
        /home/travis/gopath/src/github.com/future-architect/vuls/oval/util.go:118 +0x399
created by github.com/future-architect/vuls/oval.getDefsByPackNameViaHTTP
        /home/travis/gopath/src/github.com/future-architect/vuls/oval/util.go:122 +0x1fa
```

# Addition Details

Well, I have a distributed infrastructure ( with several kubernetes cluster ), in my docker-host I'm running _vuls_ as _binary_ or _container_, _cve_ and _oval_ databases are running as servers.

cve: go-cve-dictionary v0.1.1 fad7cfc
oval: goval-dictionary  dca4f21ry

## Expected Behavior

Create full-text format report

## Actual Behavior

The report hasn't created.

## Steps to reproduce the behaviour

To send the _report_ from 

```
./vuls report -format-full-text -log-dir=./logs -cvedb-url=http://vulsdb:1323 -refresh-cve -ovaldb-url=http://vulsdb:2323 -debug
```
But, I only get that error, into my clusters infrastructure, because, in local, it works correctly, even using a ssh-tunnel from my laptop to my externals _vulsdb_ databases, as well as, if I use an local sqlite database, it works correctly ( in docker-hosts ).

--- a/oval/util.go
+++ b/oval/util.go
@@ -109,15 +109,14 @@ func getDefsByPackNameViaHTTP(r *models.ScanResult) (
 				NewVersionRelease: pack.FormatVer(),
 				isSrcPack:         false,
 			}
-			for _, pack := range r.SrcPackages {
-				reqChan <- request{
-					packName:        pack.Name,
-					binaryPackNames: pack.BinaryNames,
-					versionRelease:  pack.Version,
-					isSrcPack:       true,
-				}
+		}
+		for _, pack := range r.SrcPackages {
+			reqChan <- request{
+				packName:        pack.Name,
+				binaryPackNames: pack.BinaryNames,
+				versionRelease:  pack.Version,
+				isSrcPack:       true,
 			}
-
 		}
 	}()
 

